repos/system_upgrade/common/actors/scan_default_initramfs/libraries/scan_default_initramfs.py

```diff
@@ -0,0 +1,39 @@
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import DefaultInitramfsInfo, DefaultSourceBootEntry
+
+
+def scan_default_initramfs():
+    default_boot_entry = next(api.consume(DefaultSourceBootEntry), None)
+    if not default_boot_entry:
+        raise StopActorExecutionError('Actor did not receive default boot entry info.')
+
+    try:
+        initramfs_info = run(['lsinitrd', '-m', default_boot_entry.initramfs_path], split=True)['stdout']
+
+    except CalledProcessError as err:
```

Comment:
Maybe should also catch `OSError`?
I am not sure right now whether if the binary is not installed it throws `CalledProcessError` or `OSError`

Reply:
if executable is not discovered it's `OSError`

Reply:
I am going to assume `lsinitrd` being always present thanks to it being a part of the `dracut` package.